integrations/hubspot-conversations/src/index.ts

```diff
@@ -0,0 +1,70 @@
+import { createIntegration, createOAuthHandler, Logger } from '@gitbook/runtime';
+import { Router } from 'itty-router';
+import { HubSpotRuntimeContext } from './types';
+import { getHubSpotOAuthConfig } from './client';
+import { ingestConversations } from './conversations';
+import { configComponent } from './config';
+import { handleWebhook, HubSpotWebhookPayload } from './webhook';
+
+const logger = Logger('hubspot-conversations');
+
+export default createIntegration<HubSpotRuntimeContext>({
+    fetch: async (request, context) => {
+        const router = Router({
+            base: new URL(
+                context.environment.spaceInstallation?.urls?.publicEndpoint ||
+                    context.environment.installation?.urls.publicEndpoint ||
+                    context.environment.integration.urls.publicEndpoint,
+            ).pathname,
+        });
+
+        /*
+         * Webhook to ingest conversations when they are closed.
+         */
+        router.post('/webhook', async (request) => {
```

Comment:
How is the webhook configured? Globally or per installation?

I don't see anything about it